apps/updownload/src/components/Import.vue

```diff
@@ -179,8 +183,14 @@ export default {
           fetch(url, options)
             .then((response) => {
               if (response.ok) {
-                response.text().then((successText) => {
-                  this.success = successText;
+                response.json().then((response) => {
+                  if (response.id) {
+                    // if identifier is present it's a task
```

Comment:
Try to not add comments like these, if you really want to make the code more clear use functions/variable names. Here you could do `if (isTask(response))`.